geo-traits/src/polygon.rs

```diff
@@ -61,8 +61,12 @@ impl<T: CoordNum> PolygonTrait for Polygon<T> {
     }
 
     fn exterior(&self) -> Option<Self::RingType<'_>> {
-        // geo-types doesn't really have a way to describe an empty polygon
-        Some(Polygon::exterior(self))
+        let ext_ring = Polygon::exterior(self);
+        if ext_ring.num_points() == 0 {
+            None
+        } else {
+            Some(ext_ring)
+        }
```

Comment:
cc @michaelkirk, this is correct semantics, right?

Reply:
That's how I understand it.